server/app.js

```diff
@@ -60,6 +60,10 @@ if (process.env.NODE_ENV === "production") {
 app.use(notFound);
 app.use(errorHandler);
 
+server.listen(8000, () => {
+  console.log("server running on port: 8000");
+});
+
```

Comment:
We have a `bin/www` file for running our server. We don't need this

Reply:
ok @moffatethan let me take that line out.